sc-kpm/scp/CMakeLists.txt

```
@@ -12,13 +11,14 @@ set(CMAKE_C_FLAGS_DEBUG "-DSCP_DEBUG")
 add_definitions(-DSCP_MATH)
 add_definitions(-DSCP_STRING)
 
-add_library (scp SHARED ${SOURCES_CPP} ${SOURCES_C} ${HEADERS})
+add_library(scp SHARED ${SOURCES_CPP} ${SOURCES_C} ${HEADERS})
+
+target_include_directories(
+    scp
```

```suggestion
target_include_directories(scp
```
Is better. Because we also can provide dependencies without modificator. In that case target name will be at the same line as command